pkg/cmd/opts/install.go

```diff
@@ -223,35 +223,61 @@ func (o *CommonOptions) InstallGlooctl() error {
 }
 
 // InstallKustomize installs kustomize
-func (o *CommonOptions) InstallKustomize() error {
+func (o *CommonOptions) InstallKustomize() (err error) {
```

Comment:
Let's not use named return values here. We don't use them otherwise and they are recommended for short functions only anyways - https://tour.golang.org/basics/7

Personally,  I stay completely clear of them